src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+            // then there's nothing left to visit anyway.
+            if (receiverOpt == null)
+            {
```

Comment:
You may also want to mention cases in which this is possible (like in a `nameof` expression)